src/index.js

```diff
@@ -71,3 +71,8 @@ exports.getAllIsoCodes = () => {
     return country.code;
   });
 }
+
+exports.getCountryByCode = (number) => {
+  let getCountryCode = number.split('-')[0]
```

Comment:
can we add the null, invalid input type check and send the response : Not a valid input